package.json

```diff
@@ -183,6 +184,7 @@
     "cypress-wait-until": "^1.7.1",
     "decompress": "^4.2.1",
     "deep-diff": "^1.0.2",
+    "docdash": "^1.2.0",
```

Comment:
I picked Docdash because it was simple and had the one feature I wanted that the default template doesn't, which is a sidebar that lists the functions within a module.